README.md

```diff
@@ -38,7 +38,15 @@ with the [hostnames](http://community.opscode.com/cookbooks/hostnames) cookbook
 
 # Testing
 
-In order to test the cookbook you will need to install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/).
+In order to test the cookbook you will need to install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/). 
+
+* Tested on:
+
+| Operating System | Vagrant version | VirtualBox version |
+|---|---|---|
+| Fedora 25 |  1.8.5 | 5.1.14r112924 |
+| OS X 10.12.2 | 1.9.1 | 5.0.32r112930 |
```

Comment:
We don't need a detailed list definitely, but there are some combinations of vagrant and vbox that are incompatible and you get meaningless errors when running the kitchen tests. We should provide at lease one or some officially supported versions.